app/services/ccms/applicant_add_requestor.rb

```diff
@@ -52,10 +58,10 @@ def contacts(xml)
     end
 
     def address(xml)
-      xml.__send__('ns4:AddressLine1', '102 Petty France')
-      xml.__send__('ns4:City', 'London')
+      xml.__send__('ns4:AddressLine1', @address.address_line_one + ' ' + @address.address_line_two)
```

Comment:
I'd suggest the combination of lines one and two of the address, is moved to `Address`, where handling on line two absence can be easily added. Something like:
```ruby
def first_lines
  [address_line_one, address_line_two].select(&:present?).join(' ')
end
```
Then here:
```ruby
xml.__send__('ns4:AddressLine1', @address.address_first_lines)
```